walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
```

Comment:
This file (and the others below) is in the directory `walksthroughs/week-3-server/...`

I think we can probably remove them from the pull request?